README.md

```diff
@@ -194,3 +205,68 @@ You can switch back to the rotating machine id by deleting the override file
 and restarting the daemon.  You can do a quick test to check that your
 machine-id has changed by running "hprobe" and verifying that a record has
 landed on your backend telemetrics server, with the specified machine id.
+
+Event Id
+----------------------
+
+This is a 32 character lowercase hexadecimal string i.e. '5de9de8d5f3c6a7d445d75ba01cc3322'.
+This header is used to group multiple records by an event id. Before this header
+every single record could have been thought of an event, however this is not always
+the case. There are "events" that trigger the creation of multiple records (i.e.
+updates). The ```event_id``` header was added for probes with the capability to
+detect events and group records based on such events. This header was added to
+```telem-record-gen``` and can be specified using the ```-e``` (```--event-id```
+long form) switch.
+
+```{r, engine=bash, count_lines}
+  -e, --event-id        Event id to use in the record
+```
+
+Debugging locally with telemetrics-client
+-----------------------------------------------
+
+The function of the telemetry-client is to report debug/crash information to
+a backend (see ```server``` key in configuration). This information is helpful
```

Comment:
s/telemetry-client/telemetrics-client

Also, I think it's misleading to state that its purpose is to report debug/crash information. This may be misinterpreted as meaning that it *only* reports this information.